frontend/src/lib/components/ProductEmptyState/ProductEmptyState.tsx

```diff
@@ -0,0 +1,72 @@
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { IconOpenInNew, IconPlus } from 'lib/lemon-ui/icons'
+import { BuilderHog3, DetectiveHog } from '../hedgehogs'
+
+export const ProductEmptyState = ({
+    productName,
+    thingName,
+    description,
+    action,
+    actionElementOverride,
+    docsURL,
+    customHog,
+}: {
+    // The name of the product, e.g. "Cohorts"
+    productName: string
+    // The name of the thing that they will create, e.g. "cohort"
+    thingName: string
+    description: string
+    // The action to take when the user clicks the CTA
+    action?: () => void
+    // If you want to provide a custom action button instead of using the default one
+    actionElementOverride?: JSX.Element
+    docsURL?: string
+    customHog?: JSX.Element
+}): JSX.Element => {
+    const actionable = action || actionElementOverride
+    return (
+        <div className="border-2 border-dashed border-border-light w-full p-8 flex justify-center rounded-md mt-6">
```

Comment:
Subjectively, I suggest using a 1px border, since we really don't use 2px anywhere in the product. Then balancing that thinness out with using the base border color instead of the light one. (Also, I'm trying to cut down on border-light usage for PostHog 3000, since we just have a lot of ways to color borders right now 😅)
And `mt-6` seems like quite a lot, I think we want to be pretty compact (but with some breathing room), so my proposal is `mt-2`.
```suggestion
        <div className="border border-dashed w-full p-8 flex justify-center rounded-md mt-2">
```

Reply:
I tend to like lots of whitespace but, fine, can go for `mt-2`. However, the thinner border doesn't look nearly as nice, even in a darker color:

![image](https://github.com/PostHog/posthog/assets/18598166/a56ca44e-22fa-4b7f-b732-7ee9e022eedf)

vs 

![image](https://github.com/PostHog/posthog/assets/18598166/d5bb1f5c-a2ec-4280-8498-c6c3ad02a842)


I feel like there are two types of borders:

1. A border that delineates something amongst other similar items or content
2. A border that is used to make something stand out as being completely different

I feel like this is the latter, and a thin border doesn't do the job. But, if you _really_ don't like it then I will use a 1px border.

Reply:
Those are good points, fair. Looking at this again, I would just like to suggest switching to the standard border color. I think that serves delineation better either way, along with simplifying things.